PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/sqlserver/common/ServerNameTextField.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ */
+package com.microsoft.azure.toolkit.intellij.sqlserver.common;
+
+import com.microsoft.azure.management.resources.Subscription;
+import com.microsoft.azure.toolkit.intellij.common.ValidationDebouncedTextInput;
+import com.microsoft.azure.toolkit.lib.common.form.AzureValidationInfo;
+import com.microsoft.azure.toolkit.lib.sqlserver.service.AzureSqlServer;
+import com.microsoft.azuretools.azurecommons.helpers.NotNull;
+import org.apache.commons.lang3.StringUtils;
+
+import java.util.Objects;
+import java.util.regex.Pattern;
+
+public class ServerNameTextField extends ValidationDebouncedTextInput {
```

Comment:
rename to `AzureSqlServerNameTextField`? As you are using the `checkNameAvailability` from `Azure.az(AzureSqlServer.class)`